config/pipeline.yaml

```diff
@@ -447,6 +447,20 @@ jobs:
         patchlevel: 7
     kcidb_test_suite: kselftest.dt
 
+  kselftest-discoverable-devices-probe:
+    template: kselftest.jinja2
+    kind: test
```

Comment:
Is this a single test case or a test suite? It [might need adjusting its `kind`](https://github.com/kernelci/kernelci-pipeline/pull/648#discussion_r1665524836); see https://github.com/kernelci/kernelci-pipeline/commit/396c67d0ba5260a56b4c4ceccd1635665150f527

Reply:
I agree with @pawiecz's suggestion. It should have `kind: job` in case of a test suite.

Reply:
It is a test suite. Fixed.